bin/prettier.js

```diff
@@ -55,10 +56,12 @@ const argv = minimist(args, {
     "range-end",
     "stdin-filepath",
     "config",
-    "resolve-config"
+    "resolve-config",
+    "ignore-path"
   ],
   default: {
-    color: true
+    color: true,
+    "ignore-path": null
```

Comment:
Sorry, might not have been clear before. This should be `"ignore-path": ".prettierignore"`, then you don't need `DEFAULT_IGNORE_PATH`.